shared/styles/style-guide-colors.js

```diff
@@ -7,6 +7,8 @@ export default {
   blue3: '#a8d7ff',
   blue4: '#e6f3ff',
 
+  blue3_40: 'rgba(168, 215, 255, 0.4)',
```

Comment:
I'll change in another PR it touches a bunch of files (64 flow errors after making the change)